2] loto/main.py

```diff
@@ -0,0 +1,79 @@
+from random import sample
+import os
+
+def print_fortune(fortune):
+    os.system("clear")
+    for i in range(0, len(fortune), 5):
+        print(*fortune[i:i+5])
+
+def select_number():
+    os.system("clear")
+#    fortune = [i for i in range(1,50)]
+    fortune = list(range(1,50))
+    select_numbers = []
+    count = 0
+
+#    while not count == 6:
+    while count < 6:
```

Comment:
opet je to velmi komplikvane ta logika. vzdyt jenom vybiras par cisel.. to nemusi a nema byt takhle prekomplikvane